monasca-alarms/monasca_alarm_definition.py

```diff
@@ -582,7 +582,7 @@ def main(args=None):
     }
 
     if not monascaclient_found:
-        print("python-monascaclient >= 1.0.9 is required", file=sys.stderr)
+        print("python-monascaclient>=1.6.0<1.7.0 is required", file=sys.stderr)
```

Comment:
Why `<1.7.0` part? And what if newer version is installed?

Reply:
Well, the idea is not use untested monascaclient, only the one that is proven to work.

Reply:
Although, I think I will remove that bit because we have this image on monasca/client which satisfies that condition.

Reply:
Can monascaclient return it's version so it could be tested?